### PR TITLE
Initial prototype of LogDB interface: include code base and associative couchapp

### PR DESCRIPTION
**Pull request by vkuznet**

### CODE REVIEW

File: src/couchapps/LogDB/validate_doc_update.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                     || matchesRole("-admin", "group:couchdb");
+
+   //---------------------------------
+   // Authorization rules for Myapp DB
+
+   // The following rule aplies for all operation types
+   var allowed = isGlobalAdm || matchesRole("admin", "group:reqmgr")
+                             || matchesRole("web-service", "group:facops")
+                             || matchesRole("production-operator", "group:dataops");
+
```
Comment (ticoann) [inline src/couchapps/LogDB/validate_doc_update.js]: || matchesRole("production-manager","group:dataops")
                             || matchesRole("t0-operator","group:dataops")
I am not sure there is still production manager role that was what we had for wmstats

**Comment by ticoann** [comment]
Valentin, could we move this under WMCore/Services? That directory contains external service apis. And rpm dependency is already set for the most applications which will use this.

**Comment by vkuznet** [comment]
Seangchan, I moved LogDB under WMCore/Services and added requested roles. Please review.

**Comment by ticoann** [comment]
Thanks Valentin, I will add the deploy script. I can add wmstats deployment script or separate application.
reqmgr2 is not a good place, since tier0 won't install reqmgr2. What do you think?

**Comment by vkuznet** [comment]
Yes, I think it should be separate application if we want to have external API
and allow add comments into LogDB from external clients (reqmgr2, data-ops
scripts, plain curl).

**Pull request merged**